### PR TITLE
Use FeePolynomial in WeightToFee implementation. 

### LINKED ISSUE #37: Use FeePolynomial instead of WeightCoefficientCalc
Since EPT is now using `polkadot-v0.9.42` we can get rid of [`WeightCoefficientCalc`](https://github.com/paritytech/extended-parachain-template/blob/36b529c1388e4775953d3ef6064b14b5c3684429/runtime/src/lib.rs#L194) and use [`FeePolynomial`](https://github.com/paritytech/substrate/blob/0bf14335fac7a8eda6c4e5324fb3fcfcba86ccf6/primitives/weights/src/lib.rs#L174-L177) in our `WeightToFee` implementation instead.

### PR DESCRIPTION
**Pull request by weezy20**
Closes #37

### CODE REVIEW

**Comment by weezy20** [comment]
I think we should hold out on merging this one before #44 is merged in order to avoid another rebase.

**Review by stiiifff — APPROVED** [review]

**Comment by stiiifff** [comment]
LGTM (this is a small one, it shouldn't be blocked by #44 which is much bigger and will take more time to review).

**Pull request merged**
